framework/logging.cpp

```diff
@@ -1453,6 +1453,10 @@ void logging_mark_knob_used(std::string_view key, TestKnobValue value, KnobOrigi
             else
                 return stdprintf("-0x%" PRIx64, -v);
         }
+        std::string operator()(double v)
+        {
+            return stdprintf("%f", double(v));
```

Comment:
I recommend "%.17g" here. Also possible, "%.*g" with a first argument of `std::numeric_limits<double>::digits10 + 2`.

Why 17 or that +2? Long story...

Reply:
Used the %.17g"  Just seems easier to read